examples/getting_started/2-Customization.ipynb

```diff
@@ -181,20 +174,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "curve_opts = dict(height=100, width=600, xaxis=None, tools=['hover'], color='red', line_width=1.5)\n",
-    "spike_opts = dict(height=100, width=600, yaxis=None, color='grey', line_width=0.25)\n",
-    "\n",
-    "curve = hv.Curve(spike_train, 'milliseconds', 'Hertz')\n",
-    "spikes = hv.Spikes(spike_train, 'milliseconds', [])\n",
-    "\n",
-    "(curve.options(**curve_opts) + spikes.options(**spike_opts)).cols(1)"
+    "dotted_options = opts.Curve(color='purple', width=600, height=250, line_dash='dotted')\n",
+    "dotted_options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Try tab-completing the options for `Curve` above or specifying an invalid keyword. The object can be passed to the `.opts` call to customize a `Curve`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dotted  = hv.Curve(spike_train, 'milliseconds', 'Hertz')\n",
+    "dotted.opts(dotted_options)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When using ``.options`` to apply options directly to an individual object we do not have to explicitly declare which object the options apply to, however often it is useful to set options on a composite object. In these cases the options can be declared as a dictionary of the type name and the options. The code below is therefore equivalent to the syntax we used above:"
+    "When working with elements, you don't have to use the `opts` utility. If you know the keywords you wish to use or have finalized a visualization and want to maximize readability for publication, you can inline the keywords instead:"
```

Comment:
Not just with elements, ``opts`` makes sense when you want to customize only the top-level object, which in most cases will be true when styling elements but also frequently will work for setting the width/height of an overlay for instance. Not sure how best to explain that though. Would be interested to see what @jbednar thinks.

Reply:
I decided to gloss over that for the reasons you mention - it is hard to explain nicely. Happy to take suggestions.

Reply:
I'm ok with glossing over that distinction here and not mentioning it.